hack/local-up-cluster.sh

```diff
@@ -695,6 +695,31 @@ function wait_node_ready(){
   fi
 }
 
+function refresh_docker_containerd_runc {
+  apt update
```

Comment:
requiring debian-ish is a pretty big departure isn't it?

Reply:
perhaps we should update these packages in the host environment instead? (IE kubekins-e2e)

Reply:
@BenTheElder this piece of code will not be executed outside of the CI